recipe/meta.yaml

```diff
@@ -1,25 +1,27 @@
 {% set name = "elastic-transport" %}
-{% set version = "8.4.0" %}
+{% set version = "8.17.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/elastic-transport-{{ version }}.tar.gz
-  sha256: b9ad708ceb7fcdbc6b30a96f886609a109f042c0b9d9f2e44403b3133ba7ff10
+  url: https://github.com/elastic/elastic-transport-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 58dd66a35cf11577e017d2ffb2d85b53523a0f3faa7f89b4b54882987afcdcb4
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
+    - wheel
   run:
-    - python >=3.6
+    - python
     - urllib3 >=1.26.2,<2
```

Comment:
```suggestion
    - urllib3 >=1.26.2,<2
    - certifi
```
https://github.com/elastic/elastic-transport-python/blob/v8.17.0/setup.py#L54C10-L54C17

Reply:
Could you add to run_constrained:
```
run_constrained:
  - httpcore<1.0.6
```
https://github.com/elastic/elastic-transport-python/blob/v8.17.0/setup.py#L69C14-L69C28

Reply:
We'll leave sphinx off for now then!

Reply:
Added the `run_constrained` dep and seems to be working just fine now that `certifi` has been added.